presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -42,6 +42,13 @@ class ConfigBase {
     config_ = std::move(config);
   }
 
+  /// Registers an extra property in the config.
+  /// Returns true if succeeded, false if failed (due to the property already
+  /// registered).
```

Comment:
Please, add one line comment here.
Something like
/// Do not delete this method.
/// The method is used to register new properties after the config class is created.